Doc/library/test.rst

```diff
@@ -1238,21 +1229,21 @@ The :mod:`test.support` module defines the following classes:
 
    A context manager to force import to return a new module reference.  This
    is useful for testing module-level behaviors, such as the emission of a
-   DeprecationWarning on import.  Example usage::
+   :exc:`DeprecationWarning` on import.  Example usage::
 
       with CleanImport('foo'):
           importlib.import_module('foo')  # New reference.
 
 
 .. class:: DirsOnSysPath(*paths)
 
-   A context manager to temporarily add directories to sys.path.
+   A context manager to temporarily add directories to :data:`sys.path`.
 
-   This makes a copy of :data:`sys.path`, appends any directories given
-   as positional arguments, then reverts :data:`sys.path` to the copied
+   This makes a copy of ``sys.path``, appends any directories given
```

Comment:
```:data:`sys.path` ```  was creating a link to it, ``` ``sys.path`` ``` won't, why changing it? (multiple occurrences following)